### PR TITLE
Single button to copy all the info

### LINKED ISSUE #59: Copy and paste the full query panel info
### Is your enhancement related to a problem? Please describe.

Currently, there is a "Copy" link per query to send to the clipboard the cURL request. It would be great if we could have a "copy" button that would copy everything: all queries and their args, body, and results. That would make things easier to share.

### Designs

_No response_

### Describe alternatives you've considered

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by MARQAS**
### Description of the Change
This PR will replace the copy cURL buttons into a single copy button that will be used to copy the whole data instead of single cURL request. The position is also changed from bottom of the cURL request to top of all the cURL calls for better visibility and usage

Closes #59 

### How to test the Change
- Install Debug bar and Debug bar ElasticPress along with ElasticPress. 
- Go to the website and search for anything after activating ElasticPress.
- Enable query logging
- From the top right corner: Click on Debug->ElasticPress
- Notice that the multiple copy button will now be replaced by single copy button
- Click on it and it will copy all the cURL requests of the page

### Changelog Entry

> Changed - Copy link per query to a single copy button to copy all cURL requests instead of individual queries. 

### Credits

Props @MARQAS 

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests pass.

### CODE REVIEW

**Review by felipeelia — CHANGES_REQUESTED** [review]
@MARQAS In addition to the git conflict we need to solve, we will need to make some changes on this one:

1. We do not want to replace the "Copy cURL Request". That is useful when someone wants to replay a specific query.
2. The button that copies everything can't be named "Copy cURL Requests" because it is copying much more than the cURL commands to replay the request. Something like "Copy Requests Info" would work better.
3. We can come up with better formatting for the copied report. Following the format we have in the Status Report would make it easier to read.

In addition to that, it would be great if the user could also *download* the debug info as a txt file. I've created an [issue in the ElasticPress repo](https://github.com/10up/ElasticPress/issues/3288) talking about that.

File: assets/css/main.css [hunk 1]
```diff
@@ -42,7 +42,91 @@
...
+#debug-menu-target-EP_Debug_Bar_ElasticPress a.button,
+#debug-menu-target-EP_Debug_Bar_ElasticPress button.ep-copy-button.button.qm-button {
+	background: #f6f7f7 !important;
+	border-color: #2271b1 !important;
+	border-radius: 3px !important;
+	border-width: 1px;
+	
+	/* Overriding some Debug Bar "important" rules */
+	color: #2271b1 !important;
+	color: #2271b1 !important;
```
Comment (burhandodhy) [inline assets/css/main.css]: Duplicate property

File: assets/js/main.js [hunk 2]
```diff
@@ -1,4 +1,17 @@
 wp.domReady(() => {
+	const copyBtn = document.querySelector('.ep-copy-button');
+	const successMessage = document.querySelector('.ep-copy-button-wrapper__success');
+	const request = copyBtn.getAttribute('data-request');
+
+	copyBtn.addEventListener('click', function () {
+		navigator.clipboard.writeText(request).then(function () {
```
Comment (burhandodhy) [inline assets/js/main.js:7]: Getting this error `Uncaught TypeError: Cannot read properties of undefined (reading 'writeText')`

File: classes/QueryFormatter.php [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+	 * Format queries to display
+	 *
+	 * @param array $queries Queries to be displayed
+	 * @return array
+	 */
+	public function format_queries_for_display( $queries ) : array {
+		$formatted_queries = [];
+
+		$labels = [
+			'wp_url'      => esc_html__( 'Page URL', 'elasticpress' ),
```
Comment (burhandodhy) [inline classes/QueryFormatter.php]: Invalid text-domain. It should be `debug-bar-elasticpress`

File: assets/css/main.css (hunk 1 above)
Comment (felipeelia) [reply assets/css/main.css]: Fixed in 9778850. Thanks @burhandodhy!

File: classes/QueryFormatter.php (hunk 3 above)
Comment (felipeelia) [reply classes/QueryFormatter.php]: Fixed in https://github.com/10up/debug-bar-elasticpress/commit/977885082578323404c91e430ab09a6f7d16a17a. Thanks @burhandodhy!

File: assets/js/main.js (hunk 2 above)
Comment (felipeelia) [reply assets/js/main.js:7]: @burhandodhy I wasn't able to reproduce this one. In which browser are you seeing this?

**Pull request merged**